libr/util/sys.c

```diff
@@ -861,7 +860,6 @@ R_API int r_sys_run(const ut8 *buf, int len) {
 	if (!pid) {
 		ret = cb ();
 		exit (ret);
-		return ret;
```

Comment:
Nope. Imagine exit fails, because the symbol is replaced or something we want to continue gracefully